core/solver/cgs.cpp

```diff
@@ -138,34 +138,42 @@ void Cgs<ValueType>::apply_impl(const LinOp *b, LinOp *x) const
     r_tld->copy_from(r.get());
 
     int iter = 0;
+    /* Memory movement summary:
+     * 27n * values + 2 * matrix/preconditioner storage
```

Comment:
two iterations

Reply:
same here: should we note that this is the memory movement for one loop containing two iterations, then average?